cpp/eudaq/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ include_directories(include)
 
 aux_source_directory(src MODULE_SRC)
 
-find_package(eudaq CONFIG REQUIRED NO_DEFAULT_PATH)
+find_package(eudaq CONFIG 2.4 REQUIRED)
```

Comment:
To prevent to link against an unwanted eudaq  installation, the system/defualt/evn_path are remvoed with flag REQUIRED NO_DEFAULT_PATH

Reply:
Fine for me - will not change anything - see next commit